libraries/app/include/graphene/app/api.hpp

```diff
@@ -185,31 +185,29 @@ namespace graphene { namespace app {
                                                                         unsigned limit = 100,
                                                                         uint32_t start = 0) const;
          /**
-          * @brief Get history of orders with b:a pair
+          * @brief Get list of orders with b:a pair
           * @param a asset in pair
           * @param b asset in pair
-          * @param limit Maximum number of orders to retrieve
+          * @param limit Maximum limit of orders to retrieve
           * @return A history of orders with b:a pair, where a <= b, otherwise a and b will be swaped
           */
          vector<order_history_object> get_fill_order_history( asset_id_type a, asset_id_type b, uint32_t limit )const;
 
          /**
-          * @brief Gets accumulated market history with b:a pair
+          * @brief Get accumulated market history with b:a pair
+          * Thus supporting OHLC https://www.investopedia.com/terms/o/ohlcchart.asp
           * @param a asset in pair
           * @param b asset in pair
           * @param bucket_seconds Represents the length in seconds of bucket
-          * @param start Time in seconds from where buckets begin
-          * @param end Time in seconds where buckets stop
-          * @return A history in market with b:a pair (maximum markets to retrieve is less than 200), 
-          * where a <= b, otherwise a and b will be swaped
+          * @param start Time point in seconds since 1970 from where buckets begin
```

Comment:
This description doesn't help much here.. You should put your feet in the user's shoes: what's the most helpful description when I'm looking at the document. Of course you need to understand the API in order to write document for it.

For this API, a user doesn't need to count how many seconds has been past since 1970.

IMHO it's better to provide an example E.G. "2018-09-12T18:00:00".